dynamic_rest/serializers.py

```diff
@@ -346,6 +361,22 @@ def get_fields(self):
         for name in deferred:
             serializer_fields.pop(name)
 
+        # Set read_only flags based on read_only_fields meta list.
+        # Here to cover DynamicFields not covered by DRF.
+        ro_fields = getattr(self.Meta, 'read_only_fields', [])
+        self.make_fields_read_only(serializer_fields, ro_fields, True)
```

Comment:
This would be more re-usable if it took a flag name parameter similar to `_get_flagged_field_names`, e.g:

```
self.flag_fields(serializer_fields, ro_fields, 'read_only', True)
```